src/main/java/Duke.java

```diff
@@ -1,10 +1,145 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        String taskType;
+        String taskName;
+        String by;
+        String at;
+        Scanner splitInputScanner;
+        String[] userInputSplitted = {""};
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        // To do textUITesting, comment line 19
+        System.out.println(logo);
+        printDividingLine();
+
+        // Print welcome message
+        printWelcomeMessage();
+
+        // Scan for input
+        Scanner userInputScanner = new Scanner(System.in);
+        String userInput = userInputScanner.nextLine();
+
+        // Separate taskType and taskName(may contain
+        // dates/time) if applicable
+        if(isOneWord(userInput)) {
+            taskType = userInput;
+            taskName = userInput;   // dummy value
+        }
+        else{
```

Comment:
Should the else statement be at the same line as the curly bracket of the if statement?